packages/extension-table/src/cell/table-cell.ts

```diff
@@ -40,6 +40,17 @@ export const TableCell = Node.create<TableCellOptions>({
           const colwidth = element.getAttribute('colwidth')
           const value = colwidth ? colwidth.split(',').map(width => parseInt(width, 10)) : null
 
+          // if there is no colwidth attribute on the cell, try to get it from the colgroup
+          if (!value) {
+            const cols = element.closest('table')?.querySelectorAll('colgroup > col')
+            const cellIndex = Array.from(element.parentElement?.children || []).indexOf(element)
```

Comment:
Can it be the case that the cellIndex is null? (for example, when the array is []). In the next line, you access `cols[cellIndex]` so I wonder what would happen if it is null.

Reply:
fixed